src/app-layer-ftp.c

```diff
@@ -434,7 +434,7 @@ static int FTPParseRequestCommand(uint8_t *input, uint32_t input_len, const FtpC
 
     *cmd_descriptor = NULL;
 
-    for (int i = 0; i < FTP_COMMAND_MAX; i++) {
+    for (int i = 0; i < FTP_COMMAND_MAX - 1; i++) {
```

Comment:
these changes should be squashed into the appropriate commit(s)

The commit message of this commit doesn't comply with our standards either.

Reply:
My bad ... You're correct, it doesn't.